lib/proto.js

```diff
@@ -63,15 +63,20 @@ function getESModule(exports) {
   if (typeof exports == 'object' || typeof exports == 'function') {
     if (getOwnPropertyDescriptor) {
       var d;
-      for (var p in exports)
-        if (d = Object.getOwnPropertyDescriptor(exports, p))
-          defineProperty(esModule, p, d);
+      for (var p in exports) {
+        try {
```

Comment:
try - catch deoptimizes the whole `getESModule` function here, so we should probably expand `normalSetProperty` to be a `tryDefineProperty`?